CHANGELOG.rst

```diff
@@ -10,6 +10,7 @@ Syntax changes and new commands
 Deprecations and removed features
 ---------------------------------
 - The ``fish_history`` value "default" is no longer an alias for "fish" (:issue:`7650`).
+- The ``__fish_prepend_sudo`` function has been removed in favor of ``__fish_prepend`` (:issue:`7905`).
```

Comment:
Should we rename it to something like `fish_commandline_prepend`?

Reply:
Yeah, that sounds good. `__fish_prepend_sudo` is an undocumented internal function and if we're going to make it a special input function it should have a non-underscored name and an entry in `bind.rst`.

Reply:
> special input function

It's a normal function because I think special input don't take arguments. So it should be added to the  `Additional functions` section in `bind.rst`